### PR TITLE
cookbook vectordb: Qdrant

### PR DESCRIPTION
**Pull request by lucifertrj**
Add a qdrant example with an Agentic rag using SqlAgentStorage for memory and WebsiteKnowledgeBase

### CODE REVIEW

File: cookbook/vectordb/qdrant_db.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+from phi.agent import Agent
+from phi.storage.agent.sqlite import SqlAgentStorage
+from phi.model.openai import OpenAIChat
+from phi.tools.duckduckgo import DuckDuckGo
+
+# get Qdrant credentials from https://cloud.qdrant.io/
+QDRANT_URL = "<endpoint-url>"
+QDRANT_API_KEY = "<cluster-key>"
+OPENAI_API_KEY = "<replace-with-your-key>" #platform.openai.com
```
Comment (manthanguptaa) [inline cookbook/vectordb/qdrant_db.py:15]: Take this from environment variables

File: cookbook/vectordb/qdrant_db.py
Comment (manthanguptaa) [inline cookbook/vectordb/qdrant_db.py:1]: I would like the example to be consistent with other vector db examples

Comment (lucifertrj) [reply cookbook/vectordb/qdrant_db.py:1]: right, will make the changes.

**Comment by lucifertrj** [comment]
@manthanguptaa done

**Review by ysolanky — APPROVED** [review]

**Pull request merged**